actions/bump-all-packages/main.ts

```diff
@@ -0,0 +1,138 @@
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+import { z } from 'zod';
+import { getArrayInput, getStringInput } from '@elementor-editor-github-actions/utils';
+import * as fs from 'fs';
+import { glob } from 'glob';
+
+const packagesDir = 'packages';
+export async function run() {
+    try {
+        const inputs = parseInputs();
+        const {
+            targetBranch,
+            targetDirectories,
+            changeType
+        } = inputs;
+
+        await core.group('Checking out target branch', async () => {
+            try {
+                await exec.exec('git', ['checkout', targetBranch]);
+                core.info(`Successfully checked out branch: ${targetBranch}`);
+            } catch (error) {
+                throw new Error(`Failed to checkout branch ${targetBranch}: ${error}`);
+            }
+        });
+
+        await core.group('Configuring git user', async () => {
+            try {
+                await exec.exec('git', ['config', 'user.name', 'GitHub Actions']);
+                await exec.exec('git', ['config', 'user.email', 'github-actions@github.com']);
+
+                core.info('Git user configured');
+            } catch (error) {
+                throw new Error(`Failed to configure git user: ${error}`);
+            }
```

Comment:
Make sure the user config the git user before this actions. just validate it in this action